src/time.c

```diff
@@ -40,13 +40,12 @@ int clock_gettime( clockid_t clk_id, struct timespec *ts )
     }
     else if ( CLOCK_MONOTONIC == clk_id || CLOCK_MONOTONIC_RAW == clk_id )
     {
-      const uint64_t clock = mach_absolute_time();
-      mach_timebase_info_data_t timebase;
-      mach_timebase_info(&timebase);
-      uint64_t tdiff = clock * timebase.numer / timebase.denom;
-      if ( CLOCK_MONOTONIC == clk_id ) {
-        tdiff = THOUSAND * ( tdiff / THOUSAND );
-      }
```

Comment:
Do we still need the check for `CLOCK_MONOTONIC`, along with the adjustment if true?

Reply:
It looks like the rounding logic was added last month, via the following commit:

https://github.com/macports/macports-legacy-support/commit/88852aa8928004f1b02e40d80d924ccf8fc31195

@cjones051073 Chris, your thoughts?

Reply:
So please put back the scaling for `CLOCK_MONOTONIC` you appear to have removed,